onnxruntime/test/providers/cpu/tensor/tensor_op_test.cc

```diff
@@ -579,7 +581,8 @@ TEST(TensorOpTest, MeanVarianceNormalizationCPUTest) {
   MeanVarianceNormalizationFunctionDefaultPerChannel();
 }
 
-TEST(TensorOpTest, ImageScalerTest) {
+// Disabled because this experimental op got removed from ONNX and causes test failure - (Commit a89a4a162f3d0c9b8269e97327c44297b04214a1 in ONNX)
```

Comment:
You must solve this.

Reply:
see: https://github.com/Microsoft/onnxruntime/blob/master/onnxruntime/core/framework/environment.cc#L57

Reply:
Thanks for this pointer. I will take a look. I created the PR based on Pranav's preliminary advice to disable them and create a task to track them.

I see Ke's PR to add the removed experimental ops as contrib ops, so do I make the corresponding kernels as contrib kernels or is my comment totally out of sync ?

Reply:
I would suggest you waiting him. After his PR get checked in, I guess this test will pass.

Reply:
Because we are going to publish a new release,  we should be very careful this release won't break existing models. If we take your this change, without Ke's change in environment.cc, then we may have such a risk.

Reply:
Ah, Ke already has a PR to resolve this. Yeah, in that case, let's wait for his PR to get merged.

Reply:
Well theoretically Ke's PR will not even pass build as he is including an ONNX commit that includes change to an op spec and the corresponding ONNX model test will not pass with the existing  implementation (unless excluded). It needs changes in this PR to comply with the spec change.

Reply:
Is this change needed by this release?

Reply:
No, it's not needed for this release and it can go into master after the release branch is cut if you are worried about the change causing instability, but we would like this to be done as early as possible as it's needed by some converted object detection models to leverage. 

Cc:@faxu

Reply:
Please sync your branch and try it again